doc/builders/images/dockertools.xml

```diff
@@ -12,7 +12,7 @@
   <title>buildImage</title>
 
   <para>
-   This function is analogous to the <command>docker build</command> command, in that it can be used to build a Docker-compatible repository tarball containing a single image with one or multiple layers. As such, the result is suitable for being loaded in Docker with <command>docker load</command>.
+   This function is analogous to the <command>docker build</command> command, in that it can be used to build a Docker-compatible repository tarball containing a single image with one or multiple layers. As such, the <literal>./result/*</literal> is suitable for being loaded in Docker with <command>docker load</command>.
```

Comment:
The output of a `dockerTools` is a docker image, a .tar.gz. Representing it as `./result/`, which implies a directory, is wrong.

Additionally, a *result* is not necessarily a symlink as produced by `nix-build`. It is a store path, which can be used as inputs to other derivations too. The word *result* does not need further qualifications as it is just the word, with a well-known definition in the context of Nix builds.

What it might benefit from, if it is possible with the documentation tooling (current or incoming) is a link to a glossary of terms. Though this circles back to the concept that the reference documentation assumes a level of familiarity with Nix.

Reply:
> The output of a dockerTools is a docker image, a .tar.gz. Representing it as ./result/, which implies a directory, is wrong.

I just learned this. This it's inconvenient - a path would be better: https://discourse.nixos.org/t/dockertools-result-not-a-folder/8771

> Additionally, a result is not necessarily a symlink as produced by nix-build. It is a store path, which can be used as inputs to other derivations too. The word result does not need further qualifications as it is just the word, with a well-known definition in the context of Nix builds.

Then it must be _italic_.

Reply:
In the context of input `docker load` it can't be a _result_ it must be one of the `./.../*.tar.gz` (a single one).

Reply:
Or... the author didn't mean _result_ but just plain English "result" - this is how I first read it.